extlib/millhone/src/cmd.rs

```diff
@@ -20,6 +20,7 @@ use typed_builder::TypedBuilder;
 use walkdir::DirEntry;
 
 pub mod analyze;
+pub mod analyze_jars;
```

Comment:
[nit, optional] i'd call this "analyze container" or something to make it more clear